### ISSUE TITLE
0.6.2.11 to 0.7.0.23

### ISSUE THREAD

**Issue by GamerWithGlasses** [marcussacana/FAR#23]
I started with 0.5.4.9 and it worked. Then I upgraded FAR from that to 0.6.2.11. It appeared to still work. the game loaded full screen. DId not have a chance to test the FAR menu with Ctrl-Alt-Backspace but will later.

My question is: i read I only need to remove dxgi.dll. Do I need to start off with 0.7.0.14 as this includes CEGUI then copy over the files from 0.7.0.23 into NiER : A folder and remove dxgi.dll? Oh also go to default_dinput8.ini and change [Steam.Log] from silent=false to silent=true?

I need to do it manually and I wanted to be sure I do this the right way. Oh, I believe I'm using bartman v3. 

Not really big issue neccesarily, it is just about going from 0.5.x.x or 0.6.x.x to 0.7.x.x.

Thanks man!

**Comment by marcussacana** [comment]
The silent=false can help if you have crash issues or are you trying use a retail kid build....
Well, try the CTRL + SHIFT + ALT + Backscape too, have some versions that this hotkey changes...
the dxgi.dll is a older build of the FAR, the dinput8.dll is to newer builds, you just can't have both in the game directory...

**Comment by GamerWithGlasses** [comment]
yeah I had that crash issue initially. Then I did the dxd9.dll fix (had a no graphics found error, something like that) then started with 0.5.4.9. which got the game to finally load then upped FAR to 0.6.2.11. But using bartman worked great. I've also read I can do select and start on the game pad.

Going to my earlier question. If I start with 0.7.0.14, I copy over the updated CEGUI folder into the game directory. Then copy over the latest files in 0.7.0.23 and delete dxgi.dll? Sound correct?

**Comment by marcussacana** [comment]
yep

**Comment by GamerWithGlasses** [comment]
Worked good thanks. The game started to crash after I completed the prologue. However, it appears if you were to set the CPU affinity in task manager -> Details -> NieRAutomata.exe and check CPU 0, 2, 4, 6 (4 core - 8 thread CPU). It greatly helps prevent the game from crashing. I was able to play for an hour with no crashing. I think there is a way to set this via command line so I don't have to do it manually.
